probe/probe.go

```diff
@@ -302,3 +334,362 @@ func probeNode(c *k8s.Client) {
+	var data [][]string
+	err := json.Unmarshal(buf, &kd)
+	if err != nil {
+		log.Println("an error occured when parsing file", err)
+		return err
+	}
+	data = append(data, []string{"\t OS Image:", green(kd.OSImage)})
+	data = append(data, []string{"\t Kernel Version:", green(kd.KernelVersion)})
+	data = append(data, []string{"\t Kubelet Version:", green(kd.KubeletVersion)})
+	data = append(data, []string{"\t Container Runtime:", green(kd.ContainerRuntime)})
+	data = append(data, []string{"\t Active LSM:", green(kd.ActiveLSM)})
+	data = append(data, []string{"\t Host Security:", green(strconv.FormatBool(kd.HostSecurity))})
+	data = append(data, []string{"\t Container Security:", green(strconv.FormatBool(kd.ContainerSecurity))})
+	data = append(data, []string{"\t Container Default Posture:", green(kd.ContainerDefaultPosture.FileAction) + itwhite("(File)"), green(kd.ContainerDefaultPosture.FileAction) + itwhite("(Capabilities)"), green(kd.ContainerDefaultPosture.NetworkAction) + itwhite("(Network)")})
+	data = append(data, []string{"\t Host Default Posture:", green(kd.HostDefaultPosture.FileAction) + itwhite("(File)"), green(kd.HostDefaultPosture.CapabilitiesAction) + itwhite("(Capabilities)"), green(kd.HostDefaultPosture.NetworkAction) + itwhite("(Network)")})
+	renderOutputInTableWithNoBorders(data)
+	return nil
+}
+
+// sudo systemctl status kubearmor
+func isSystemdMode() bool {
+	cmd := exec.Command("systemctl", "status", "kubearmor")
+	_, err := cmd.CombinedOutput()
+	if err != nil {
+		return false
+	}
+	color.Green("\nFound KubeArmor running in Systemd mode \n\n")
+	return true
+}
+
+func probeSystemdMode() error {
+	jsonFile, err := os.Open("/tmp/karmorProbeData.cfg")
+	if err != nil {
+		log.Println(err)
+		return err
+	}
+
+	buf, err := io.ReadAll(jsonFile)
+	if err != nil {
+		log.Println("an error occured when reading file", err)
+		return err
+	}
+	_, err = boldWhite.Printf("Host : \n")
+	if err != nil {
+		color.Red(" Error")
+	}
+	err = printKubeArmorProbeOutput(buf)
+	if err != nil {
+		log.Println("an error occured when printing output", err)
+		return err
+	}
+	return nil
+}
+
+func getAnnotatedPods(c *k8s.Client) error {
+	// Annotated Pods Description
+	var data [][]string
+	pods, err := c.K8sClientset.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	policyMap, err := getPoliciesOnAnnotatedPods(c)
+	if err != nil {
+		color.Red(" Error getting policies on annotated pods")
+	}
+
+	for _, p := range pods.Items {
+		if p.Annotations["kubearmor-policy"] == "enabled" {
+			v, err := c.K8sClientset.CoreV1().Pods(p.Namespace).Get(context.Background(), p.Name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			for key, value := range v.Labels {
+				for x, y := range policyMap {
+					if slices.Contains(y, key+":"+value) {
+						data = append(data, []string{v.Namespace, v.Name, x})
+					}
+
+				}
+			}
+		}
+	}
+	_, err = boldWhite.Printf("Annotated pods : \n")
```

Comment:
```suggestion
	_, err = boldWhite.Printf("Armored Up pods : \n")
```

We can fancy this up :) Essentially we are saying what pods are being/can be protected by KubeArmor.